TES backend fails citing s3 filesystem error

Hi,
I am trying to execute the example found in the rabix-1.0.5 release

Running it using LOCAL backend works as expected

```
./rabix examples/dna2protein/dna2protein.cwl.json examples/dna2protein/inputs.json --tes-url http://localhost:8000
```


When using funnel locally installed and I specify  the `backend.embedded.types=TES` in `core.properties` I get the following error coming from rabix:

```
./rabix --tes-url http://localhost:8000  examples/dna2protein/dna2protein.cwl.json examples/dna2protein/inputs.json --tes-url http://localhost:8000
[2019-01-14 15:33:25.067] [ERROR] Encountered an error while starting local backend.
org.rabix.engine.service.BootstrapServiceException: com.google.inject.ProvisionException: Unable to provision, see the following errors:

1) Error injecting constructor, java.nio.file.FileSystemNotFoundException: S3 filesystem not yet created. Use newFileSystem() instead
  at org.rabix.backend.tes.service.impl.LocalTESStorageServiceImpl.<init>(LocalTESStorageServiceImpl.java:36)
  while locating org.rabix.backend.tes.service.impl.LocalTESStorageServiceImpl
  at org.rabix.backend.tes.TESModule.configure(TESModule.java:30) (via modules: org.rabix.cli.BackendCommandLine$1 -> org.rabix.backend.tes.TESModule)
  while locating org.rabix.backend.tes.service.TESStorageService
    for field at org.rabix.backend.tes.service.impl.LocalTESWorkerServiceImpl.storage(LocalTESWorkerServiceImpl.java:81)
  while locating org.rabix.backend.tes.service.impl.LocalTESWorkerServiceImpl

1 error
	at org.rabix.engine.service.impl.BootstrapServiceImpl.start(BootstrapServiceImpl.java:47) ~[rabix-cli.jar:na]
	at org.rabix.cli.BackendCommandLine.main(BackendCommandLine.java:398) ~[rabix-cli.jar:na]
Caused by: com.google.inject.ProvisionException: Unable to provision, see the following errors:

1) Error injecting constructor, java.nio.file.FileSystemNotFoundException: S3 filesystem not yet created. Use newFileSystem() instead
  at org.rabix.backend.tes.service.impl.LocalTESStorageServiceImpl.<init>(LocalTESStorageServiceImpl.java:36)
  while locating org.rabix.backend.tes.service.impl.LocalTESStorageServiceImpl
  at org.rabix.backend.tes.TESModule.configure(TESModule.java:30) (via modules: org.rabix.cli.BackendCommandLine$1 -> org.rabix.backend.tes.TESModule)
  while locating org.rabix.backend.tes.service.TESStorageService
    for field at org.rabix.backend.tes.service.impl.LocalTESWorkerServiceImpl.storage(LocalTESWorkerServiceImpl.java:81)
  while locating org.rabix.backend.tes.service.impl.LocalTESWorkerServiceImpl

1 error
	at com.google.inject.internal.Errors.throwProvisionExceptionIfErrorsExist(Errors.java:486) ~[rabix-cli.jar:na]
	at com.google.inject.internal.MembersInjectorImpl.injectMembers(MembersInjectorImpl.java:67) ~[rabix-cli.jar:na]
	at com.google.inject.internal.InjectorImpl.injectMembers(InjectorImpl.java:987) ~[rabix-cli.jar:na]
	at org.rabix.engine.service.impl.BackendServiceImpl.scanEmbedded(BackendServiceImpl.java:82) ~[rabix-cli.jar:na]
	at org.rabix.engine.service.impl.BootstrapServiceImpl.start(BootstrapServiceImpl.java:45) ~[rabix-cli.jar:na]
	... 1 common frames omitted
Caused by: java.nio.file.FileSystemNotFoundException: S3 filesystem not yet created. Use newFileSystem() instead
	at com.upplication.s3fs.S3FileSystemProvider.getFileSystem(S3FileSystemProvider.java:280) ~[rabix-cli.jar:na]
	at com.upplication.s3fs.S3FileSystemProvider.getPath(S3FileSystemProvider.java:298) ~[rabix-cli.jar:na]
	at java.nio.file.Paths.get(Paths.java:143) ~[na:1.8.0_172]
	at org.rabix.backend.tes.service.impl.LocalTESStorageServiceImpl.<init>(LocalTESStorageServiceImpl.java:47) ~[rabix-cli.jar:na]
	at org.rabix.backend.tes.service.impl.LocalTESStorageServiceImpl$$FastClassByGuice$$148bf36a.newInstance(<generated>) ~[rabix-cli.jar:na]
	at com.google.inject.internal.DefaultConstructionProxyFactory$FastClassProxy.newInstance(DefaultConstructionProxyFactory.java:89) ~[rabix-cli.jar:na]
	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:111) ~[rabix-cli.jar:na]
	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:90) ~[rabix-cli.jar:na]
	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:268) ~[rabix-cli.jar:na]
	at com.google.inject.internal.FactoryProxy.get(FactoryProxy.java:56) ~[rabix-cli.jar:na]
	at com.google.inject.internal.ProviderToInternalFactoryAdapter$1.call(ProviderToInternalFactoryAdapter.java:46) ~[rabix-cli.jar:na]
	at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1092) ~[rabix-cli.jar:na]
	at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40) ~[rabix-cli.jar:na]
	at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:194) ~[rabix-cli.jar:na]
	at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:41) ~[rabix-cli.jar:na]
	at com.google.inject.internal.SingleFieldInjector.inject(SingleFieldInjector.java:54) ~[rabix-cli.jar:na]
	at com.google.inject.internal.MembersInjectorImpl.injectMembers(MembersInjectorImpl.java:132) ~[rabix-cli.jar:na]
	at com.google.inject.internal.MembersInjectorImpl$1.call(MembersInjectorImpl.java:93) ~[rabix-cli.jar:na]
	at com.google.inject.internal.MembersInjectorImpl$1.call(MembersInjectorImpl.java:80) ~[rabix-cli.jar:na]
	at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1085) ~[rabix-cli.jar:na]
	at com.google.inject.internal.MembersInjectorImpl.injectAndNotify(MembersInjectorImpl.java:80) ~[rabix-cli.jar:na]
	at com.google.inject.internal.MembersInjectorImpl.injectMembers(MembersInjectorImpl.java:62) ~[rabix-cli.jar:na]
	... 4 common frames omitted


```

Rabix version: 1.0.5
Funnel version 0.8.0 
Funnel has been tested with cwl-tes and works as expected.
Thanks in advance for your help
K

--- a/rabix-cli/config/core.properties
+++ b/rabix-cli/config/core.properties
@@ -53,7 +53,8 @@ tes.postprocessing_thread_pool=1
 #if gs is used, authentication is taken from the environment
 #for s3 the following keys should be added to a config file: s3.access_key, s3.secret_key
 
-tes.storage_base=s3://endpoint/bucket/folder
+tes.storage_base=s3://s3.amazonaws.com/<your-bucket>
+#tes.storage_base=file:///your/local/path
 
 #S3 options.
 #Format: s3.<provider_name>.<option>
@@ -68,7 +69,7 @@ s3.amazon.max_retry_error=10
 s3.amazon.connection_timeout=50000
 s3.amazon.max_connections=50
 s3.amazon.socket_timeout=50000
-s3.amazon.signer_override=AWS4SignerType
+s3.amazon.signer_override=AWSS3V4SignerType
 s3.amazon.path_style_access=false
 
 s3.ceph.endpoints= s3://cephhost:8888/

--- a/rabix-cli/src/main/java/org/rabix/cli/BackendCommandLine.java
+++ b/rabix-cli/src/main/java/org/rabix/cli/BackendCommandLine.java
@@ -404,13 +404,13 @@ public class BackendCommandLine {
         commonInputs = bindings.translateToCommon(inputs);
         if (inputsFile != null) {
           final Path finalInputs = inputsFile;
-          FileValueHelper.updateFileValues(commonInputs, (FileValue f) -> {
-            fixPaths(finalInputs, f);
-            if (f.getSecondaryFiles() != null)
-              for (FileValue sec : f.getSecondaryFiles()) {
+          FileValueHelper.updateFileValues(commonInputs, (FileValue file) -> {
+            fixPaths(finalInputs, file);
+            if (file.getSecondaryFiles() != null)
+              for (FileValue sec : file.getSecondaryFiles()) {
                 fixPaths(finalInputs, sec);
               }
-            return f;
+            return file;
           });
         }
       } catch (BindingException e1) {

